pkg/transforms/compression.go

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright (c) 2017 Cavium
```

Comment:
@rsdmike Why is Cavium event mentioned in here? Isn't all of the app-functions-sdk contributed by Intel?

Reply:
Well, I didn't really change the code they wrote for compression and encryption (in a different pull request), I only changed the function signature, return values, and input handling. So it didn't seem correct to remove their copyright.